build.assets/Dockerfile

```diff
@@ -1,51 +1,16 @@
 # This Dockerfile makes the "build box": the container used to build
 # official releases of Teleport and its documentation
-FROM debian:jessie
+FROM quay.io/gravitational/buildbox-base
```

Comment:
can you add a tag or sha hash, this will make the build reproducible.

Reply:
also, it's teleport buildbox base, can you add this to the name to make it clear?

Reply:
@klizhentas actually I was hoping this to be a base image for _any_ build process. That image does not contain Golang or anything at all, really. It's just a small base image we can start using as a base for our builds.